coordinatorlib/src/main/resources/com.socrata.datacoordinator.truth.schema/20150615-add-table-secondary-groups-config.xml

```diff
@@ -0,0 +1,25 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<databaseChangeLog xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+                   xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+                   xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog
+    http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-2.0.xsd">
+    <changeSet author="krah" id="20150615-add-column-secondary-group">
+        <preConditions onFail="MARK_RAN">
+            <not>
+                <tableExists tableName="secondary_groups_config"/>
+            </not>
+        </preConditions>
+        <createTable tableName="secondary_groups_config">
+            <column name="group_id" type="VARCHAR(40)">
```

Comment:
I'm tempted to suggesting calling that group_name throughout instead of id since it isn't really an id, but that is a bit of just a nit perhaps.